src/services/storage/models/stake.model.ts

```diff
@@ -3,13 +3,15 @@ import BigNumber from 'bignumber.js'
 
 import { BigNumberStringType } from '../../../sequelize'
 
+export const SUPPORTED_TOKENS_SYMBOLS = ['rbtc', 'rif', 'cfp']
```

Comment:
This is definitely place to have it. Either `config` or `definitions.ts`, as already mentioned I prefer the config with the token's address.